influxdb/rootfs/Dockerfile

```diff
@@ -22,22 +18,15 @@ RUN \
   adduser -S -s /bin/bash -G influxdb influxdb
 
 RUN \
-  apk add --update bash wget git mercurial go && \
-  mkdir -p $GOPATH/src/github.com/influxdata && \
-  cd $GOPATH/src/github.com/influxdata && \
-  git clone https://github.com/influxdata/influxdb.git && \
-  cd influxdb && \
-  git checkout -b $INFLUX_VERSION && \
-  go get -t ./... && \
-  go build ./... && \
-  go install ./... && \
-  mkdir -p $INFLUXDB_HOME && \
-  cp $GOPATH/bin/influx* $INFLUXDB_HOME/   && \
-  chown -R influxdb:influxdb $INFLUXDB_HOME && \
-  mkdir -p /data /logs /config && \
-  chown -R influxdb:influxdb /data /logs /config && \
-  apk del --purge wget git mercurial go  && \
-  rm -rf /var/cache/apk/* /tmp/* /var/tmp/* $GOPATH
+  mkdir -p /usr/local/bin/ && \
+  apk update && \
+  apk add ca-certificates wget curl bash && \
```

Comment:
maybe clear the apk cache here? iirc, you can pass `--purge` to `apk` and it'll do it for you